### PR TITLE
Update rootfs-and-kernel-setup.md

### PR DESCRIPTION
**Pull request by Pupkur**
Use fallocate instead of dd to speed up creation of big files.
Comparision:
time dd if=/dev/zero of=./rootfs.ext4 bs=1M count=10000
10000+0 records in
10000+0 records out
10485760000 bytes (10 GB, 9.8 GiB) copied, 35.4951 s, 295 MB/s
real	0m35.516s
user	0m0.050s
sys	0m11.860s

time fallocate -l 10G rootfs.ext4
real	0m0.008s
user	0m0.001s
sys	0m0.005s

Issue #, if available:

Description of changes:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by Pupkur** [comment]
Done.

**Comment by andreeaflorescu** [comment]
@Pupkur please also squash the commits. It should only be one commit.

File: docs/rootfs-and-kernel-setup.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ To build an EXT4 image:
    on how much data you'll want to fit inside:
    
    ```bash
-   dd if=/dev/zero of=/rootfs.ext4 bs=1M count=50
+   fallocate -l 50mb rootfs.ext4
```
Comment (dhrgit) [inline docs/rootfs-and-kernel-setup.md]: There's a small inaccuracy here: the doc says "50 MiB", but the command allocates 50 MB (i.e. 50 million bytes). Can you please update the command to use MiB (i.e. 50 * 1024 * 1024 bytes)?

**Comment by dianpopa** [comment]
@Pupkur are you still interested in merging this PR?

**Comment by dianpopa** [comment]
Closing this due to inactivity.
@Pupkur feel free to reopen it whenever.

**Pull request closed without merging**